src/main/java/org/folio/consortia/controller/ErrorHandlingController.java

```diff
@@ -84,6 +91,11 @@ public Errors handleTokenErrors(Exception e) {
   @ResponseStatus(HttpStatus.BAD_REQUEST)
   @ExceptionHandler(PrimaryAffiliationException.class)
   public Errors handlePrimaryAffiliationException(Exception e) {
+    return createExternalError(e.getMessage(), VALIDATION_ERROR);
+  }
+  @ResponseStatus(HttpStatus.BAD_REQUEST)
+  @ExceptionHandler(PublicationException.class)
```

Comment:
PublicationException now thrown for both cases when user does not have primary affiliation or when tenant list is empty, maybe it makes sense to introduce error codes for initiate PublicationException constructor with them